clusters/build-clusters/common/ci-operator/admin_ci-operator_rbac.yaml

```diff
@@ -8,7 +8,7 @@ items:
     name: ci-operator
     namespace: ci
   imagePullSecrets:
-  - name: regcred
+  - name: registry-pull-credentials
```

Comment:
This is also usage, please leave this out.

Reply:
New commit just adds this as a new entry.

Reply:
I don't undestand how "New commit just adds this as a new entry." resolves what I said. I'm pretty convinced this should not be here - it will race. What if we start using the new secret before they are actually created?

Reply:
removed.